lib/validations.js

```diff
@@ -208,6 +220,17 @@ validations.isURL = function isURL(paramName, options, customMessage) {
   );
 }
 
+validations.isPlainObject = function isPlainObjectValidator(paramName, customMessage) {
```

Comment:
i'm afraid that the function can't be called `isPlainObject`, if you look in the function i'm calling the `isPlainObject` function from lodash, if the name of the function is `isPlainObject` it would never reach the `isPlainObject` function from lodash